README.md

```
@@ -136,10 +191,37 @@ echo "password" | envault dump --password-stdin
+envault unset --help
+envault dump --help
 
 # バージョン情報を表示
 envault version
+# または
+envault --version
+```
+
+### コマンド構造
+
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)